### PR TITLE
Features/publisher subscriber

### PR DESCRIPTION
**Pull request by hamzajg**
Stream Subscriber can subscribe to Stream Publisher..

### CODE REVIEW

**Comment by VaughnVernon** [comment]
@hamzajg Is this ready for review and merge?

/cc @tjaskula

**Comment by hamzajg** [comment]
@VaughnVernon yes this is ready, still WIP stream processor, and some PubSub behaviors, will create new PR when done.

**Comment by tjaskula** [comment]
I'll be reviewing that today

**Comment by tjaskula** [comment]
@hamzajg I merged the #2 and maybe I shouldn't? I wanted to merge that one but It needs to resolved conflicts.

**Comment by hamzajg** [comment]
Thanks for #2 , no problem, will resolve conflicts..

**Comment by tjaskula** [comment]
@hamzajg Any progress on it?

**Comment by hamzajg** [comment]
@tjaskula Still WIP for some PubSub test cases and StreamProcessor..

**Comment by tjaskula** [comment]
> StreamProcessor



> @tjaskula Still WIP for some PubSub test cases and StreamProcessor..

Ok, waiting for your green light. BTW, I've added appveyor builds to check on the build status as well and run UT.

File: src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (tjaskula) [inline src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj]: Why do you need that ?

**Review by tjaskula — COMMENTED** [review]
Minor comments

File: src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj (hunk 1 above)
Comment (hamzajg) [reply src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj]: Yeah, I get a build failed from AppVeyor suggested `please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.`

Comment (tjaskula) [reply src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj]: It should be something else. We don't need it on other projects. Please remove it I'll be investigating

Comment (hamzajg) [reply src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj]: Yeah, I compared with the other projects and it's not used, I will remove it..

Comment (tjaskula) [reply src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj]: I've added the necessary configuration it should build without the namespace. Please merge it into this PR

Comment (hamzajg) [reply src/Vlingo.Xoom.Streams.Tests/Vlingo.Xoom.Streams.Tests.csproj]: OK, thanks..

**Comment by tjaskula** [comment]
@hamzajg I've pushed fixes to **master** to fix builds https://github.com/vlingo-net/xoom-net-streams/runs/3265784056?check_suite_focus=true.

We're not ready yet to support C#9 preview features like target-typed object creation `private readonly StringBuilder _builder = new()` or nullable generics `T?`.

**Comment by tjaskula** [comment]
Closing PR because it's not the latest up to date

**Pull request closed without merging**